tilelang/autotuner/tuner.py

```
@@ -424,17 +434,412 @@ def _load_result_from_disk(self, key) -> AutotuneResult:
+                continue
+            if item is None:
+                break
+            jit_kernel, config, idx = item
+            try:
+                if timeout > 0:
+                    call_result_queue: queue.SimpleQueue = queue.SimpleQueue()
+
+                    def _run_benchmark_target():
+                        try:
+                            latency, worker_ref_latency = benchmark_target(
+                                jit_kernel=jit_kernel,
```

Ruff (0.15.12)

[warning] Function definition does not bind loop variable `jit_kernel`

(B023)